http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -686,15 +679,28 @@ class DataprocInterpreter[F[_]: Timer: Parallel: ContextShift](
       F.liftIO(IO.fromFuture(IO(googleDirectoryDAO.removeMemberFromGroup(groupEmail, memberEmail)))),
       when409
     )
+    val poll = for {
+      ctx <- ev.ask
+      d: DoneCheckable[Boolean] = (_: Boolean) == addToGroup
+      isMember <- Timer[F].sleep(5 seconds) >>
```

Comment:
is it necessary to wait 5 seconds? just wonder if the group addition usually happens relatively fast

Reply:
Probably not.. actually it doesn't seem like polling the group addition fixes the problem so I probably will remove it. I am not sure where the delay is actually happening causing the Dataproc SA to sometimes not have permission to the compute image.

The only other thing I can think of is retrying the Dataproc cluster creation, but that's tricky because it's an asynchronous error. But maybe it's doable in this case..